test/functional/Jsr292/src/com/ibm/j9/jsr292/api/MethodHandleAPI_foldArguments.java

```diff
@@ -30,11 +30,12 @@
 import static java.lang.invoke.MethodType.methodType;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import org.openj9.test.util.VersionCheck;
 
 public class MethodHandleAPI_foldArguments {
 	final Lookup lookup = lookup();
 	final String MHClassName = "java.lang.invoke.MethodHandles";
-
+	final String internalFoldArgumentMethod = (VersionCheck.major() >= 12) ? "foldArgumentsWithCombiner" : "foldArguments";
```

Comment:
I think you can just to `VersionCheck.major() > 11`

Reply:
jdk12 is obsolete

Reply:
Agreed and updated as suggested.